bcipy/display/paradigm/matrix/display.py

```diff
@@ -187,11 +203,12 @@ def build_grid(self) -> Dict[str, visual.TextStim]:
         grid = {}
         for sym in self.symbol_set:
             pos_index = self.sort_order(sym)
+            pos = self.positions[pos_index]
             grid[sym] = visual.TextStim(win=self.window,
                                         text=sym,
                                         color=self.grid_color,
                                         opacity=self.start_opacity,
-                                        pos=self.positions[pos_index],
+                                        pos=pos,
```

Comment:
Is this the position in psychopy?

Reply:
Yes, the `visual.TextStim` is a psychopy display component, so the position information is expected to be a valid psychopy format, depending on the units. When no units are provided it uses the default 'norm' units.